docs/source-pytorch/upgrade/sections/1_4_regular.rst

```diff
@@ -0,0 +1,47 @@
+.. list-table:: reg. user 1.4
+   :widths: 40 40 20
+   :header-rows: 1
+
+   * - If
+     - Then
+     - Ref
+
+   * - relied on the ``outputs`` in your  ``LightningModule.on_train_epoch_end`` or ``Callback.on_train_epoch_end`` hooks
+     - rely on either ``on_training_epoch_end`` or set outputs as attributes in your ``LightningModule`` instances and access them from the hook
+     - #7339
```

Comment:
as follow-up: can we add links to these?

Reply:
yes